aiida/restapi/translator/calculation/__init__.py

```diff
@@ -91,10 +92,11 @@ def __init__(self, **kwargs):
         }
 
     @staticmethod
-    def get_files_list(dirobj, files=None, prefix=None):
+    def get_files_list(nobj, dirobj=None, files=None, prefix=None):
```

Comment:
```suggestion
    def get_files_list(node_obj, dir_obj=None, files=None, prefix=None):
```
the `n` prefix usually stands for a number, `node_obj` makes the content more clear

Reply:
I agree, it's more clear using `node_obj` and `dir_obj`. I changed it, thanks!